locales/zh-TW.json

```diff
@@ -0,0 +1,67 @@
+{
+  "projectName": {
+    "message": "請輸入項目名稱："
+  },
+  "shouldOverwrite": {
+    "dirForPrompts": {
+      "current": "目前目錄：",
+      "target": "目標目錄："
+    },
+    "message": "非空，是否覆蓋？"
+  },
+  "packageName": {
+    "message": "請輸入套件名稱：",
+    "invalidMessage": "無效的 package.json 名稱"
+  },
+  "needsTypeScript": {
+    "message": "是否使用 TypeScript 語法？"
+  },
+  "needsJsx": {
+    "message": "是否啟用 JSX 支援？"
+  },
+  "needsRouter": {
+    "message": "是否引入 Vue Router 進行單頁應用開發？"
+  },
+  "needsPinia": {
+    "message": "是否引進 Pinia 用於狀態管理？"
+  },
+  "needsVitest": {
+    "message": "是否引入 Vitest 用於單元測試"
+  },
+  "needsE2eTesting": {
+    "message": "是否要引入一款端對端（End to End）測試工具？",
+    "selectOptions": {
```

Comment:
There is a `hint` key here:
```json
"hint": "- Use arrow-keys. Return to submit.",
```

Reply:
It appears this Change Request still prevent the merging. Could you please resolve this?